rascal-lsp/src/main/rascal/demo/lang/pico/LanguageServer.rsc

```diff
@@ -81,31 +104,73 @@ Summary picoSummarizer(loc l, start[Program] input, PicoSummarizerMode mode) {
     return s;
 }
 
+@synopsis{Finds a declaration that the cursor is on and proposes to remove it.}
+list[Command] picoActions([*_, IdType x, *_, start[Program] program]) 
+    = [removeDecl(program, x, title="remove <x>")];
+
+default list[Command] picoActions(list[value] _) = [];
+
 set[loc] lookupDef(loc _, start[Program] input, Tree cursor) =
     { d.src | /IdType d := input, cursor := d.id};
 
-
+@synsopsis{Defines three example commands that can be triggered by the user (from a code lens, from a diagnostic, or just from the cursor position)}
 data Command
-  = renameAtoB(start[Program] program);
+  = renameAtoB(start[Program] program)
+  | changeToFix(loc place, str replacement)
+  | removeDecl(start[Program] program, IdType toBeRemoved)
+  ;
 
-rel[loc,Command] picoLenses(start[Program] input) = {<input@\loc, renameAtoB(input, title="Rename variables a to b.")>};
+@synopsis{Adds an example lense to the entire program.}
+rel[loc,Command] picoLenses(start[Program] input) 
+    = {<input@\loc, renameAtoB(input, title="Rename variables a to b.")>};
 
 
+@synopsis{Generates inlay hints that explain the type of each variable usage.}
```

Comment:
while I appreciate these synopsis, the amount of changes in this file makes it harder to review the actual changes versus the comment & layout changes. I would appreciate it to separate that into a different PR next time, where possible.

Reply:
ah. because this new synopsis is not related to the current PR. I see. thanks